tests/run-make/print-request-help-stable-unstable/rmake.rs

```diff
@@ -0,0 +1,31 @@
+//! For issue-138698
```

Comment:
Nit: can you elaborate *what* this test is testing, and then put the issue as a backlink? E.g.

```rs
//! Check that unstable print requests are omitted from help if compiler is in stable channel.
//!
//! Issue: <https://github.com/rust-lang/rust/issues/138698>
```